src/v3/test/integration/__snapshots__/authenticator-verification-select-authenticator.test.tsx.snap

```diff
@@ -619,6 +620,7 @@ exports[`authenticator-verification-select-authenticator renders form 1`] = `
                               Phone
                             </h3>
                             <p
+                              aria-label="+1 XXX-XXX-5310."
```

Comment:
Is it OK if the phone number on the label here gets the period at the end, too? Want to make sure that screenreaders read this correctly and doesn't end up turning it into a decimal or something.